docs/plugins/test_deploy.md

```diff
@@ -104,13 +104,10 @@ Once your package is listed on [PyPI] (and includes the `Framework :: napari`
 [classifier]), it will also be visible on the [napari
 hub](https://napari-hub.org/). To ensure you are providing the relevant metadata and
 description for your plugin, see the following documentation in the [napari hub
-GitHub](https://github.com/chanzuckerberg/napari-hub/tree/main/docs)’s docs
```

Comment:
I would suggest removing the entire sentence starting from `To ensure...` and references to the hub guide until we align on metadata sources and the hub guide is updated to reflect this.

Reply:
FWIW, I was just learning more about plugin creation when I created this PR, and wouldn't want to be the one removing references to the hub guide (or in other words, I don't usually contribute to documentation, but am encouraged to submit small fixes when I notice any opportunities).

An alternative could be to mention that metadata sources are subject to change and that discussions to align them are ongoing. (And maybe that's something the hub guide could also be updated to say!)

Reply:
@kandarpksk afaik the hub docs and preview page are currently being updated to highlight the preferred sources of metadata, so I think it's ok to update these links here, especially since the current links are invalid. I'm going to resolve this conversation.